src/dependency-core/Core/Dependency.01/Obsolete/Obsolete.Create.cs

```diff
@@ -0,0 +1,16 @@
+#nullable enable
+
+using System;
+
+namespace PrimeFuncPack
+{
+    partial class Dependency<T>
+    {
+        [Obsolete(ObsoleteMessage.OneDependencyCreateIsObsolete, false)]
```

Comment:
Hereinafter:
Let's move `false` into smth like `ObsoleteError.DependencyCreate` (single for all the Create - from one to eight based).
To make changing the mode from `false` to `true` in v1.3 or later.

Reply:
Done